plat/nvidia/tegra/soc/t132/plat_psci_handlers.c

```diff
@@ -132,7 +132,7 @@ int tegra_soc_pwr_domain_off(const psci_power_state_t *target_state)
 
 int tegra_soc_pwr_domain_suspend(const psci_power_state_t *target_state)
 {
-#if DEBUG
+#if ENABLE_ASSERTIONS
```

Comment:
The change of DEBUG to ENABLE_ASSERTIONS is needed here because PR #906 added support to enable assertions in release builds as well as debug builds. 
And so to include auxiliary code that is only used for assertions (the local cpu in this case) we can no longer use DEBUG - we need to use another define that is in sync with when assertions are enabled, rather than aligning with the build-type; ENABLE_ASSERTIONS is the that new define.